Makefile

```
@@ -119,7 +119,7 @@ build.version:
 	@echo "$(VERSION)" > $(OUTPUT_DIR)/version
 
 build.common: export SKIP_GEN_CRD_DOCS=true
-build.common: build.version helm.build mod.check crds gen-rbac
+build.common: build.version helm.build mod.check crds gen-rbac helm.dependency.clean
```

Let's not clean the dependencies after every build. When building locally, if the chart has already been downloaded, no reason to re-download it for every local build. It will be nice to be able to build offline also, if we already have the chart dependency.
```suggestion
build.common: build.version helm.build mod.check crds gen-rbac
```

---

Instead, let's add `helm.dependency.clean` to run with the `clean` action in the Makefile line 207.

---

added